.github/workflows/upload-nightly-release.yml

```
@@ -2,8 +2,8 @@ name: Build nightly PennyLane releases for TestPyPI
 
 on:
   schedule:
-    # Run every weekday at 5:50 EDT (cron is in UTC)
-    - cron: "50 9 * * 1-5"
+    # Run Tuesday-Saturday at 00:00 EST (cron is in UTC)
```

```suggestion
    # Run Tuesday-Saturday at 01:00 EDT (cron is in UTC)
```

Just to keep us consistent in docs with the nightly CIs in Catalyst and the benchmarks (which use EDT)